WORKSPACE

```diff
@@ -2,9 +2,17 @@ workspace(name = "io_bazel_rules_closure")
 
 load("//closure/private:java_import_external.bzl", "java_import_external")
 load("//closure:repositories.bzl", "closure_repositories")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 closure_repositories()
 
+http_archive(
+    name = "bazel_skylib",
+    sha256 = "bbccf674aa441c266df9894182d80de104cabd19be98be002f6d478aaa31574d",
+    strip_prefix = "bazel-skylib-2169ae1c374aab4a09aa90e65efe1a3aad4e279b",
+    urls = ["https://github.com/bazelbuild/bazel-skylib/archive/2169ae1c374aab4a09aa90e65efe1a3aad4e279b.tar.gz"],
```

Comment:
When you use a commit hash, please add the commit date in a comment.

Reply:
Resolved. I have updated to released version 0.6.0.